### PR TITLE
Updating readme to include node installed through nvm

### PR DESCRIPTION
**Pull request by rohannair**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,6 +195,13 @@ Visit http://app.hub.dev in your browser (you may need to type in the "http://")
 
 ### Optional for bonus points
 
+##### Homebrew Cask
+Homebrew cask lets you package manage your common OS X Applications
+```
+brew install caskroom/cask/brew-cask
+brew cask install google-chrome sublime-text3 iterm
```
Comment (SeeThruHead) [inline README.md]: where's atom, vim, emacs? firefox?

Comment (rohannair) [inline README.md]: Hehehe, i was being opinionated.

Comment (SeeThruHead) [inline README.md]: also does brew isntall iterm install iterm2?

**Comment by rohannair** [comment]
@SeeThruHead vim and emacs wouldn't be installed through homebrew-cask, but rather, homebrew itself.

**Pull request closed without merging**